source/_integrations/cielo_home.markdown

```diff
@@ -0,0 +1,176 @@
+---
+title: Cielo Home
+description: Instructions on how to integrate Cielo smart thermostats in Home Assistant.
+ha_category:
+  - Climate
+ha_iot_class: Cloud Polling
+ha_release: "2026.1"
+ha_config_flow: true
+ha_codeowners:
+  - '@cielo-connect'
+  - '@ihsan-cielo'
+
+ha_domain: cielo_home
+ha_platforms:
+  - climate
+ha_integration_type: integration
+ha_quality_scale: bronze
+---
+
+The **Cielo Home** {% term integration %} integration allows you to control [Cielo](https://www.cielowigle.com) 
+devices directly from Home Assistant, using the cloud-based **Cielo Connect API**.
+
+## Supported devices
+
+The **Cielo Home** {% term integration %} supports the following devices.
+
+- **Cielo Breez Max**: Smart thermostat for mini-splits.
+- **Cielo Breez Edge Pro**: Smart thermostat for mini-splits.
+- **Cielo Thermostat Eco**: Smart thermostat for central HVAC systems.
+- **Cielo Smart Thermostat**: Smart thermostat for central HVAC systems.
+
+## Prerequisites
+
+Before integrating the Cielo device/s with Home Assistant, ensure that: 
+
+- Your Cielo device is already registered in the **Cielo Home** app. 
+- The device is in supported devices 
+- The device has a configured remote (automatically or manually). 
+- Your Home Assistant server has internet access. 
+- Before proceeding, ensure the Cielo device is fully compatible with Home Assistant.
+
+### Important API Key Rules
+
+To safeguard security and prevent unauthorized access, Cielo enforces the following API security restrictions:
+
+- Each API key can be used only once. 
+- A new API key cannot be generated until the current one is revoked. 
+- You may generate a maximum of 3 keys per month. 
+- Once a new API key is generated, wait 60 seconds for it to be displayed on the screen.
+- Revoking an API key will disconnect Home Assistant immediately or within about a minute.
+
+### Obtaining an API key
+
+**Retrieving API key:** `Device [name of device] not correctly registered with remote on Sensibo cloud.`
+
+1. Visit the [Cielo Web Portal](https://home.cielowigle.com)
+2. Open the Menu by clicking the ☲ icon at the top left side.
```

Comment:
```suggestion
2. Open the Menu by selecting the ☲ icon at the top left side.
```
We prefer [select over click](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click)